addons/temps/temps.lua

```diff
@@ -0,0 +1,348 @@
+--[[
+temps v1.0
+
+Copyright © 2017, Mojo
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright
+notice, this list of conditions and the following disclaimer in the
+documentation and/or other materials provided with the distribution.
+* Neither the name of timestamp nor the
```

Comment:
For some reason it removed my other comment... below it says "Giuliano Riccio", replace that with your name (either real or alias as above, preferably the same name as in the opening line).